### ISSUE TITLE
ephemeral images

### ISSUE THREAD

**Issue by ValenArb** [jagrosh/DiscordTextHost#7]
cannot read apps that has ephemeral-attachments
Example of said link:
https://cdn.discordapp.com/ephemeral-attachments/980166362074865664/980166408514191390/Bios.txt

**Comment by jagrosh** [comment]
Yeah this is due to the difference in the base URL. I don't want to break existing functionality, so this might be best solved by having an additional query parameter that uses a different base URL (for example, `&ephemeral=true`).
